app/assets/stylesheets/group.css

```diff
@@ -4,7 +4,7 @@
 }
 
 .cont-main,
-.cont-trans  {
+.cont-trans {
   cursor: pointer;
   width: 90%;
   display: flex;
```

Comment:
- [ ] The official font face for this application based on the design given is Proxima Nova Bold for headers and Proxima Nova Light for the general text of the application. Kindly apply these design rules as adherence to design rules make you a better developer. 👍 😎  

![image](https://github.com/yegonkimutai/budget-app/assets/77106962/b261f95a-51c3-4348-b2aa-c399585dd108)